Automattic/wp-calypso
# White glove: Show offer page

#### Changes proposed in this Pull Request

This PR is a part of the larger PR that implements the white glove A/B test. Please check https://github.com/Automattic/wp-calypso/pull/41618 for more context.

* This PR implements the white glove offer page, and shows the offer to users who select a free plan in signup. 
* Note that this PR is only for implementing the offer page, so the Buy Now CTA will add only a Business plan for now(instead of a "white glove" product).

<img src="https://user-images.githubusercontent.com/1269602/83154714-17e20280-a11e-11ea-9afe-cde6611b16a3.png">

#### Testing instructions



**Scenario 1** (verify A/B test assignment works)

1. Set your locale to EN and go through the signup flow. by selecting a free plan and free domain.
2. You'll either land in Customer Home or see the offer page. In both cases, check that the AB test assignment has put you either in control or variation. Type `localstorage.ABTests;` in your browser console, you should see an entry for `whiteGloveUpsell `.
3. Set your locale to non-EN and go through signup flow. Verify that you don't get assigned to the test if you select a free plan and free domain.

**Scenario 2** (verify offer page UI)

1. Put yourself in the variant of the `whiteGloveUpsell` AB test and go through the signup flow.
2. Select a free plan and free domain and verify that you see the offer page.
3. Verify that the UI checks out with the [Figma](https://www.figma.com/file/G52TF9nObKGWqwPfDx3Wym/White-Gloves-Interstitial?node-id=50%3A307).
4. Verifythat the UI checks out in mobile screen sizes.
